lib/davinci_dtr_test_kit/profiles/task/task_validation.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../validation_test'
+
+module DaVinciDTRTestKit
+  class TaskValidationTest < Inferno::Test
+    include DaVinciDTRTestKit::ValidationTest
+
+    title 'Task resources returned during previous tests conform to the CDex Task'
+    description %(
+This test verifies resources returned from the first read conform to
+the [CDex Task](http://hl7.org/fhir/us/davinci-cdex/StructureDefinition/cdex-task-attachment-request).
```

Comment:
for CDex, the version should be 2.0.0: https://hl7.org/fhir/us/davinci-cdex/STU2/...